src/components/BlockThreeColumnHero.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import flattenImageData from 'utils/flattenImageData';
+import get from 'utils/get';
+
+import { ContentfulMedia } from 'models';
+
+import { Image } from 'components/base';
+import { Markdown } from 'components/base';
+
+const BlockThreeColumnHero = props => {
+  const fields = get(props, 'block.fields');
+  const header = get(fields, 'header', '');
+  const headerDescription = get(fields, 'headerDescription', '');
+  const headerImage = flattenImageData(get(fields, 'headerImage', {}));
+  const paragraph = get(fields, 'paragraph', '');
+  const marginBottom = get(fields, 'marginBottom', 0);
+  const marginTop = get(fields, 'marginTop', 0);
+
+  return (
+    <div
+      style={{
+        marginBottom: `${marginBottom}rem`,
+        marginTop: `${marginTop}rem`
+      }}
+      className="BlockThreeColumnHero flex flex-col"
+    >
+      <div className="flex flex-col md:flex-row col-8 mxauto px1 py1 md:pb3 justify-between">
+        {header && (
+          <h1 className="BlockThreeColumnHero__header paragraph md:col-4">
+            {header}
+          </h1>
+        )}
+        <div className="none md:flex col-4 justify-between justify-center items-center">
+          {headerDescription && (
+            <Markdown
+              className="BlockThreeColumnHero__header-description tiny"
+              src={headerDescription}
+            />
+          )}
+          {headerImage && (
```

Comment:
an empty object will return true, you should check `headerImage.url` instead

Reply:
Ooh thank you!